utils/installer/install.sh

```diff
@@ -187,18 +246,19 @@ function __attempt_to_install_with_cargo() {
     echo "Installing missing Rust dependency with cargo"
     cargo install "$1"
   else
-    echo "[WARN]: Unable to find fd. Make sure to install it to avoid any problems"
+    echo "[WARN]: Unable to find cargo. Make sure to install it to avoid any problems"
+    exit 1
   fi
 }
 
 # we try to install the missing one with cargo even though it's unlikely to be found
 function install_rust_deps() {
-  if ! command -v fd &>/dev/null; then
-    __attempt_to_install_with_cargo "fd-find"
-  fi
-  if ! command -v rg &>/dev/null; then
-    __attempt_to_install_with_cargo "ripgrep"
-  fi
+  local -A deps=(["fd"]="fd-find" ["rg"]="ripgrep")
```

Comment:
This will sadly now work on macOS which uses an ancient bash by default. It's quite unfortunate because I was also using these nice associative arrays and had to change them.

Reply:
fixed in 2d28c78